creator/ingest_runs/tasks/ingest_run.py

```diff
@@ -31,28 +31,36 @@ def run_ingest(ingest_run_uuid=None):
     are_gwo = all(check_gwo(version) for version in versions)
     if are_gwo:
         # If FEAT_INGEST_GENOMIC_WORKFLOW_OUTPUTS is enabled, try to run
-        # the custom ingest process for genomic workflow manifests. If it's not
-        # enabled, raise an error and nothing else happens.
+        # the custom ingest process for genomic workflow manifests. If it's
+        # not enabled, raise an error and nothing else happens.
         if not settings.FEAT_INGEST_GENOMIC_WORKFLOW_OUTPUTS:
-            ingest_run.fail()
-            ingest_run.save()
-            raise Exception(
-                "Ingesting genomic workflow output manifests is not enabled."
+            feature_msg = (
+                "Ingesting genomic workflow output manifests is not enabled. "
                 "Make sure that FEAT_INGEST_GENOMIC_WORKFLOW_OUTPUTS is set."
             )
+            ingest_run.error_msg = feature_msg
```

Comment:
I'm wondering if there's a better way to do this. So instead of setting the error message in all the places in the code where an exception is about to be raised, could the error message just be set in the top level ingest process exception handler using the exception object? No idea what this will give us so I'll leave it up to you to figure out, but something like this? 

```
except Exception as e:
    logger.error(str(e))
    ingest_run.error_msg(str(e))
    ingest_run.fail()
    ingest_run.save()
```

The same approach could be used in the validation task too.